dist/index.js

```diff
@@ -9727,9 +9734,14 @@ function shouldRun() {
 
 
 
-if (shouldRun()) {
-  run();
+
+
+if (!shouldRun()) {
+  core.notice("source branch matched the exclude pattern, exiting...");
```

Comment:
nit: to be precise in this function you don't know what the reason for not running the action is, but okay not a big deal

Reply:
Yea I thought so too. tbh I wanted to convert this whole action to use `actions/github-script` and organise a bit better. But perhaps later.